io.catenax.iot_sensor_data/1.0.0/IoTSensorData.ttl

```diff
@@ -0,0 +1,275 @@
+    bamm-c:unit unit:decibel.
+:TransmissionMethodCharacteristic a bamm:Characteristic;
+    bamm:preferredName "Transmission Method Characteristic"@en;
+    bamm:description "Characteristic that describes the property transmission method"@en;
+    bamm:dataType xsd:string.
+:AnalogSensorCharacteristic a bamm:Characteristic;
+    bamm:preferredName "Analog Sensor 1-10 Characteristic"@en;
+    bamm:description "Characteristic that describes the property analog sensor"@en;
+    bamm:dataType xsd:float.
+:GeoDataEntity a bamm:Entity;
+    bamm:properties ([
+  bamm:property :latitude;
+  bamm:optional "true"^^xsd:boolean
+] [
+  bamm:property :longitude;
+  bamm:optional "true"^^xsd:boolean
+] [
+  bamm:property :altitude;
+  bamm:optional "true"^^xsd:boolean
+] [
+  bamm:property :geoDataTimestamp;
+  bamm:optional "true"^^xsd:boolean
+]);
+    bamm:preferredName "Geo Data Entity"@en;
+    bamm:description "Encapsulates the Geo Data relevant attributes"@en.
+:latitude a bamm:Property;
+    bamm:preferredName "Latitude"@en;
+    bamm:description "The angle between zenith and a plane parallel to the equator.\n"@en;
+    bamm:characteristic :LatitudeCharacteristic;
+    bamm:exampleValue "4807.03"^^xsd:decimal;
+    bamm:see <https://www.wikidata.org/wiki/Q34027>.
+:longitude a bamm:Property;
+    bamm:preferredName "Longitude"@en;
+    bamm:description "geographic coordinate that specifies the east-west position of a point on the Earth's surface\n"@en;
+    bamm:characteristic :LongitudeCharacteristic;
+    bamm:exampleValue "01131.02"^^xsd:decimal;
+    bamm:see <https://www.wikidata.org/wiki/Q36477>.
+:altitude a bamm:Property;
+    bamm:preferredName "Altitude"@en;
+    bamm:description "height in relation to a certain reference point; (general) distance measurement, usually in the vertical or \"up\" direction, between a reference datum and a point or object. The reference datum also often varies according to the context\n"@en;
+    bamm:characteristic :AltitudeCharacteristic;
+    bamm:exampleValue "280.2"^^xsd:decimal;
+    bamm:see <https://www.wikidata.org/wiki/Q190200>.
+:geoDataTimestamp a bamm:Property;
+    bamm:preferredName "Geo Data Timestamp"@en;
+    bamm:description "The timestamp of the latest sensor reading of the geo data."@en;
+    bamm:characteristic bamm-c:Timestamp;
+    bamm:exampleValue "2023-02-04T14:48:54"^^xsd:dateTime.
+:LatitudeCharacteristic a bamm:Characteristic;
+    bamm:preferredName "Latitude Characteristic"@en;
+    bamm:dataType xsd:decimal;
+    bamm:see <https://en.wikipedia.org/wiki/ISO_6709>.
+:LongitudeCharacteristic a bamm:Characteristic;
+    bamm:preferredName "Longitude Characteristic"@en;
+    bamm:dataType xsd:decimal;
+    bamm:see <https://en.wikipedia.org/wiki/ISO_6709>.
+:AltitudeCharacteristic a bamm:Characteristic;
+    bamm:preferredName "Altitude Characteristic"@en;
+    bamm:dataType xsd:decimal;
+    bamm:see <http://www.nmea.de/nmea0183datensaetze.html#rmc%C2%A0>.
```

Comment:
Where is the relation between altitude and the referenced page?

Reply:
The link was wrong. Changed to http://www.nmea.de/nmea0183datensaetze.html#gga.